src/components/PopupWithForm.js

```diff
@@ -23,10 +36,28 @@ export class PopupWithForm extends Popup {
     super.setEventListeners();
 
     this.form.addEventListener("submit", (evt) => {
+      //
+      renderLoading(true, this.form);
+
       evt.preventDefault();
 
       this.handleFormSubmit(this._getInputValues());
       this.close();
     });
   }
+
+  //
+  setDeleteListener() {
```

Comment:
Please, remove `setDeleteListener` from universal `PopupWithForm`

There should **not** be any code in universal classes that doesn’t suit every popup.


According to the check list you need to create `ConfirmationPopup`